docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+
+Now that you have wired your ADC and Moisture Sensor, make sure that SPI is enabled on your Pi.
+
+SSH into your Pi and run the following command:
+
+``` shell
+sudo raspi-config
+```
+
+Once the `raspi-config` interface is open, navigate to **Interface Options**:
+
+![Raspi-config Tool interface with Interface Options selected.](../../img/plant-watering-pi/interface-options.png)
+
+Then, click on **SPI**:
+
+![Raspi-config Tool interface with SPI selected.](../../img/plant-watering-pi/spi.png)
+
+Now, click **Yes** to enable SPI:
+
+![Raspi-config Tool interface with Yes selected for SPI enablement.](../../img/plant-watering-pi/spi-enabled.png)
+
+Finally, click **Finish** and restart your Pi.
+
+### Test your Soil Moisture readings on your Pi
+
+Next, install the Adafruit ADC library [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/) on your Pi.
+
+First, make sure you have installed the required dependencies:
+
+- [Adafruit CircuitPython](https://github.com/adafruit/circuitpython)
+- [Bus Device](https://github.com/adafruit/Adafruit_CircuitPython_BusDevice)
+
+If you do not have these, download the Adafruit library and driver bundle to your Pi by following the instructions [here](https://github.com/adafruit/Adafruit_CircuitPython_Bundle).
+
+Run the following command while connected to your Pi with SSH to install [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/):
+
+``` shell
+pip3 install adafruit-circuitpython-mcp3xxx
+```
+
+Create a new directory for your plant watering robot's files and navigate to this directory in your terminal session.
+For example, run the following commands:
+
+``` shell
+mkdir plant-watering-robot
+cd plant-watering-robot
+```
+
+After navigating to this directory, make a new Python file called `adctesting` and open up the file.
+For example, run the following commands:
+
+``` shell
+touch adctesting.py
+vim adctesting.py
+```
+
+Now, add the following Python code to `adctesting.py` to test reading values from your capacitive soil moisture sensor through your MCP3008 ADC:
+
+``` python
+import time
+import busio
+import digitalio
+import board
+import adafruit_mcp3xxx.mcp3008 as MCP
+from adafruit_mcp3xxx.analog_in import AnalogIn
+
+# create the spi bus
+spi = busio.SPI(clock=board.SCK, MISO=board.MISO, MOSI=board.MOSI)
+
+# create the cs (chip select)
+cs = digitalio.DigitalInOut(board.D25) # TODO: Need to check this: example was D5. Documentation is confusing.
+
+# create the mcp object
+mcp = MCP.MCP3008(spi, cs)
+
+# create an analog input channel on pin 0
+chan = AnalogIn(mcp, MCP.P0)
+
+print('Reading MCP3008 values, press Ctrl-C to quit...')
+while True:
+    print('Raw ADC Value: ', chan.value)
+    time.sleep(1)
+```
+
+Run the code as follows:
+
+``` shell
+sudo python3 adctesting.py
+```
+
+When running this code, you should see the Moisture Sensor values outputted by the MCP3008 in the range of `0` to `1023`.
```

Comment:
```suggestion
When running this code, you should see the moisture sensor values from the MCP3008 in the range of `0` to `1023`.
```

That's a weird range that might be worth commenting on if you know why it is that but if now - leave it for now since this is urgent